frontend/staff-dashboard/src/components/flexiblepricing/statusmodal.tsx

```diff
@@ -113,15 +113,15 @@ export const FlexiblePricingStatusModal: React.FC<IFlexiblePriceStatusModalProps
             </p>
             <p>
                 <strong>Income USD:</strong>
-                <div>{modaldata.income_usd}</div>
+                <div>{modaldata.income.income_usd}</div>
```

Comment:
This (and Original income below it) should use the `formatIncome` function.

Reply:
Sure. Actually, I Intentionally left these. Wasn't mentioned to change the modal.